script.js

```diff
@@ -0,0 +1,55 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const convertToRoman = (num) => {
+    const romanNumerals = [
+      { value: 1000, numeral: 'M' },
+      { value: 900, numeral: 'CM' },
+      { value: 500, numeral: 'D' },
+      { value: 400, numeral: 'CD' },
+      { value: 100, numeral: 'C' },
+      { value: 90, numeral: 'XC' },
+      { value: 50, numeral: 'L' },
+      { value: 40, numeral: 'XL' },
+      { value: 10, numeral: 'X' },
+      { value: 9, numeral: 'IX' },
+      { value: 5, numeral: 'V' },
+      { value: 4, numeral: 'IV' },
+      { value: 1, numeral: 'I' },
+    ];
+
+    let result = '';
+    for (let i = 0; i < romanNumerals.length; i) {
```

Comment:
- Your loop runs unlimited time as you did not change the array order in your loop try this line instead:
  - Copy this line and paste it to your code to solve the issue.
```javascript
for (let i = 0; i < romanNumerals.length; i++) {
```

- In here I added `i++` to change the array loop order while it runs to + 1